docs/content/tutorial/setup.mdx

```diff
@@ -37,13 +24,38 @@ To verify that your Dagster installation was successful, it's time for you to cr
 To create the project, run:
 
 ```bash
-dagster project scaffold --name tutorial-project
+dagster project from-example --example tutorial --name tutorial-project
```

Comment:
```suggestion
dagster project from-example --example tutorial --name my-tutorial-project
```
super nit - since we have two args, it may help to name it something `my-` to tell this can be any arbitrary name.